br/pkg/task/restore.go

```diff
@@ -271,16 +282,16 @@ func CheckNewCollationEnable(
 	backupNewCollationEnable string,
 	g glue.Glue,
 	storage kv.Storage,
-	CheckRequirements bool,
+	checkCollation bool,
 ) error {
 	if backupNewCollationEnable == "" {
-		if CheckRequirements {
+		if checkCollation {
 			return errors.Annotatef(berrors.ErrUnknown,
-				"NewCollactionEnable not found in backupmeta. "+
-					"if you ensure the NewCollactionEnable config of backup cluster is as same as restore cluster, "+
-					"use --check-requirements=false to skip")
+				"NewCollationEnable not found in backupmeta. "+
+					"if you ensure the config 'new_collations_enabled_on_first_bootstrap' of backup cluster is as same as restore cluster, "+
+					"use --check-collation=false to skip")
```

Comment:
--check-collation and --check-new-collations-enable should be unified.
I think we should ask @IANTHEREAL  to make sure this config name.